### PR TITLE
l10n: pt_PT: v2.33.0 round 2

### PR DESCRIPTION
**Pull request by alexandre1985**

### CODE REVIEW

**Comment by jiangxin** [comment]
Errors found in `po/pt_PT.po`:

```$ git-po-helper check-po pt_PT
INFO Checking syntax of po file for "Portuguese - Portugal" 
ERROR Fail to check "po/pt_PT.po": exit status 1   
ERROR   po/pt_PT.po:23330: 'msgid' and 'msgstr' entries do not both end with '\n' 
ERROR   msgfmt: found 1 fatal error                 
ERROR   2798 translated messages, 1258 fuzzy translations, 1174 untranslated messages. 
```

**Comment by alexandre1985** [comment]
thank you @jiangxin .
good to go.

File: po/pt_PT.po [hunk 1]
```diff
@@ -1,5 +1,4 @@
 # Portugal's Portuguese translations for Git package.
-osgid "set compression level"
```
Comment (jiangxin) [inline po/pt_PT.po]: Why introduce this line in heading comments?

Comment (jiangxin) [reply po/pt_PT.po]: Make sure to have a valid "po/pt_PT.po" file in each commit.

Comment (alexandre1985) [reply po/pt_PT.po]: I believe it is `git-po-helper` who is doing this automatically. It is the second time now.
I will fix this

Comment (alexandre1985) [reply po/pt_PT.po]: it was already corrected in the last commit.
What should I do?

Comment (alexandre1985) [reply po/pt_PT.po]: I did a `rebase -i` and corrected/amended that commit.
Please fetch my new changes

**Comment by alexandre1985** [comment]
now it is clean

**Pull request merged**